event-stream-service/charts/event-stream-service/values-test.yaml

```
@@ -0,0 +1,16 @@
+ess:
+  nginx:
+    route:
+      ingressPrefix: stream-test
+      ingressSuffix: .apps.silver.devops.gov.bc.ca
+
+nats:
+  container:
+    merge:
+      resources:
+        limits:
+          cpu: 250m
+          memory: 256Mi
+        requests:
+          cpu: 10m
+          memory: 32Mi
```

We should be able to drop these:
```suggestion
        limits:
          cpu: 20m
          memory: 256Mi
        requests:
          cpu: 10m
          memory: 32Mi
```